src/diffusers/models/unet_2d_condition.py

```diff
@@ -261,6 +266,20 @@ def __init__(
         else:
             self.class_embedding = None
 
+        if time_embedding_act_fn is not None:
+            if act_fn == "swish":
+                self.time_embed_act = lambda x: F.silu(x)
+            elif act_fn == "mish":
+                self.time_embed_act = nn.Mish()
+            elif act_fn == "silu":
+                self.time_embed_act = nn.SiLU()
```

Comment:
Can't we do?

```py
if act_fn in ["swish", "silu"]:
    self.time_embed_act = nn.SiLU()
```

Reply:
Yes I would hope we could :) This is how it's done in a few other places in the code base so I'd like to leave it this way for now and do a follow up including a refactor of all the dispatches to the different activation functions